src/core/integrate.cpp

```diff
@@ -142,14 +142,11 @@ bool integrator_step_1(ParticleRange &particles) {
     throw std::runtime_error("Unknown value for integ_switch");
   }
   return false;
-#ifdef STOKESIAN_DYNAMICS
-  printf("SD env variable set");
-#endif
 }
 
 /** Calls the hook of the propagation kernels after force calculation */
 void integrator_step_2(ParticleRange &particles) {
-  // extern BrownianThermostat brownian; //Carl removed that because it caused him error
+  extern BrownianThermostat brownian;
```

Comment:
Yeah, sorry for these kind of changes. I figured out why this caused segfault and fixed it. Those lines are back in the code already.
What do you mean by squashing?

Reply:
Don't get me wrong, these changes are fine on a local branch. They shouldn't appear in a remote branch, though. Squashing means fusing together the commit where the comment appeared and the commit where the comment disappeared, such that the comment disappears from the git history. This can be done using a `git rebase` operation. You can find a visual depiction of squashing operations in my [2019-11-04 Group Seminar](https://gitlab.icp.uni-stuttgart.de/talk/2019-11-04-groupseminar/-/blob/master/slides.pdf), where "interleaved commits" means temporary commits that add something and then remove it, and where "fixup" is a special case of "squash".